### PR TITLE
✨ marcuscodes.me

### PR DESCRIPTION
**Pull request by geenva**

### CODE REVIEW

File: websites/M/marcuscodes.me/dist/metadata.json
Comment (Bas950) [inline websites/M/marcuscodes.me/dist/metadata.json:13]: upload to imgur

File: websites/M/marcuscodes.me/dist/metadata.json
Comment (Bas950) [inline websites/M/marcuscodes.me/dist/metadata.json:11]: v1.0.0

File: websites/M/marcuscodes.me/presence.ts
Comment (Bas950) [inline websites/M/marcuscodes.me/presence.ts:6]: merge consts

File: websites/M/marcuscodes.me/dist/metadata.json
Comment (Bas950) [inline websites/M/marcuscodes.me/dist/metadata.json:17]: tags cannot have spaces

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by geenva** [comment]
> upload proof of it working

https://i.imgur.com/HOpJiOC.png

File: websites/M/marcuscodes.me/presence.ts
Comment (Bas950) [inline websites/M/marcuscodes.me/presence.ts:5]: Merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/marcuscodes.me/presence.ts
Comment (geenva) [reply websites/M/marcuscodes.me/presence.ts:5]: sorry for my stupidity, what does that mean

Comment (ririxidev) [reply websites/M/marcuscodes.me/presence.ts:5]: e.g.: ```const d = f(),
g = h();```

Comment (geenva) [reply websites/M/marcuscodes.me/presence.ts]: i hope this works

Comment (ririxidev) [reply websites/M/marcuscodes.me/presence.ts]: image
you didnt change anything

File: websites/M/marcuscodes.me/dist/metadata.json
Comment (ririxidev) [inline websites/M/marcuscodes.me/dist/metadata.json:13]: this should be banner or promo img of site.

File: websites/M/marcuscodes.me/dist/metadata.json
Comment (ririxidev) [inline websites/M/marcuscodes.me/dist/metadata.json:12]: Change this logo to something that is not only circle with red colors.

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/M/marcuscodes.me/presence.ts
Comment (Bas950) [inline websites/M/marcuscodes.me/presence.ts]: double check if it still works with this change

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/marcuscodes.me/presence.ts
Comment (geenva) [reply websites/M/marcuscodes.me/presence.ts]: yes

File: websites/M/marcuscodes.me/dist/metadata.json
Comment (geenva) [reply websites/M/marcuscodes.me/dist/metadata.json:12]: That's the website logo.

Comment (geenva) [reply websites/M/marcuscodes.me/dist/metadata.json:12]: Should I change it to something else other than the favicon?

Comment (Bas950) [reply websites/M/marcuscodes.me/dist/metadata.json:12]: I mean... yes... a red dot aint really a logo imo

**Comment by ririxidev** [comment]
After talking with @Bas950 we decided to not merge this presence since it is low quality one. If you want more details just mention us/or me/or bas.

**Pull request closed without merging**
